web/frontend/src/components/slam-2d-render.vue

```diff
@@ -270,21 +205,14 @@ const createGridHelper = (
   return gridHelper;
 };
 
-const updateOrRemoveDestinationMarker = async () => {
+const updateOrRemoveDestinationMarker = () => {
   if (props.destVector && props.destExists) {
-    const marker = scene.getObjectByName('DestinationMarker') ??
-      await makeMarker(destMarkerUrl, 'DestinationMarker', destinationMarkerScalar);
-    marker.position.set(
-      props.destVector.x + destinationMarkerOffset.x,
-      props.destVector.y + destinationMarkerOffset.y,
-      props.destVector.z + destinationMarkerOffset.z
-    );
+    destMarker.visible = true;
+    destMarker.position.copy(props.destVector).add(destinationMarkerOffset);
```

Comment:
Why does this need to copy the `props.destVector`?

Reply:
Isn't `destinationMarkerOffset` a constant? Why do we need to add it?

Reply:
This is just using `THREE`'s built in [vector methods](https://threejs.org/docs/?q=vector3#api/en/math/Vector3.add) now to be more idiomatic.